core/tee/tee_fs_common.c

```diff
@@ -1696,6 +1702,25 @@ int tee_fs_common_rename(const char *old, const char *new)
 
 	/* finally, link the meta file, rename operation completed */
 	TEE_ASSERT(meta_filename);
+
+	/*
+	 * TODO: This prevents memory leakage at previous strdup() if
```

Comment:
Can we just simply rewrite this sentence slightly? Since it does not prevent from memory leak, but instead it will give you a warning telling you about the memory leak.